src/editors/playgroundController.ts

```diff
@@ -303,28 +346,49 @@ export default class PlaygroundController {
         }
       }
 
-      const result: ExecuteAllResult = await this.evaluateWithCancelModal();
+      const evaluateResponse: ExecuteAllResult = await this.evaluateWithCancelModal();
 
-      if (!result) {
-        this._outputChannel.clear();
+      this._outputChannel.clear();
+      if (evaluateResponse.outputLines) {
+        for (const line of evaluateResponse.outputLines)
+          this._outputChannel.appendLine(line.content);
```

Comment:
So `console.log()` etc. go to the output channel now but not the actual result? That seems like a very nice way to separate the two :+1:

Reply:
We were considering different approaches during design sync, but everyone liked this one the most :)